debian/changelog

```diff
@@ -1,3 +1,9 @@
+kolibri-server (0.3.7~beta1-0ubuntu3) UNRELEASED; urgency=medium
```

Comment:
Noting the UNRELEASED part here :+1:

Reply:
Yes, it's on purpose. I prefer to maintain it that way until we do a new release.

Reply:
Change logs that are instantly updated are great :+1: